bricks/flutter_template/hooks/post_gen.dart

```diff
@@ -2,60 +2,96 @@ import 'dart:io';
     print('Running flutter pub get');
     await Process.run('flutter', ['pub get']);
     print('Finished running flutter pub get');
+
     _setCurrentDirectory('android');
-    print('Creating android debug key...');
-    await Process.run('keytool', [
-      '-genkey',
-      '-dname',
-      "cn=Monstarlab, ou=Monstarlab, o=Monstarlab, l=Prague, s=Czech, c=CZ",
-      '-keypass',
-      '123456',
-      '-storepass',
-      '123456',
-      '-v',
-      '-keystore',
-      'debug_key.jks',
-      '-alias',
-      '{{#snakeCase}}{{applicationName}}{{/snakeCase}}_debug',
-      '-keyalg',
-      'RSA',
-      '-keysize',
-      '2048',
-      '-validity',
-      '10000'
-    ]);
-    print('Android debug key created');
-    print('Creating android release key...');
-    await Process.run('keytool', [
-      '-genkey',
-      '-dname',
-      "cn=Monstarlab, ou=Monstarlab, o=Monstarlab, l=Prague, s=Czech, c=CZ",
-      '-keypass',
-      '123456',
-      '-storepass',
-      '123456',
-      '-v',
-      '-keystore',
-      'release_key.jks',
-      '-alias',
-      '{{#snakeCase}}{{applicationName}}{{/snakeCase}}_release',
-      '-keyalg',
-      'RSA',
-      '-keysize',
-      '2048',
-      '-validity',
-      '10000'
-    ]);
-    print('Android release key created');
+
+    print('Generating android release properties');
+    _generateAndroidReleasePropertiesFile();
+    print('Android release properties generated');
+
+    print('Generating android debug key...');
+    await _generateAndroidDebugKey();
+    print('Android debug key generating');
+
+    print('Generating android release key...');
+    await _generateAndroidReleaseKey();
+    print('Android release key generating');
+
+    print('Finished setting up project');
   } catch (e) {
     throw e;
   }
 }
 
+void _generateAndroidReleasePropertiesFile() {
+  final now = DateTime.now();
+  final applicationId = '{{applicationName}}';
+
+  final password = '${now.toString()}$applicationId';
+
+  final passwordHash = password.hashCode;
+
+  File('release_key.properties').writeAsString('''
+storePassword=$passwordHash
+keyPassword=$passwordHash
+keyAlias={{#snakeCase}}{{applicationName}}{{/snakeCase}}_release
+storeFile=../release_key.jks
+''');
+}
+
+Future<void> _generateAndroidReleaseKey() async {
+  await Process.run('keytool', [
+    '-genkey',
+    '-dname',
+    "cn=Monstarlab, ou=Monstarlab, o=Monstarlab, l=Prague, s=Czech, c=CZ",
+    '-keypass',
+    '123456',
+    '-storepass',
+    '123456',
+    '-v',
+    '-keystore',
+    'release_key.jks',
+    '-alias',
+    '{{#snakeCase}}{{applicationName}}{{/snakeCase}}_release',
+    '-keyalg',
+    'RSA',
+    '-keysize',
+    '2048',
+    '-validity',
+    '10000'
+  ]);
+}
+
+Future<void> _generateAndroidDebugKey() async {
+  await Process.run('keytool', [
+    '-genkey',
+    '-dname',
+    "cn=Monstarlab, ou=Monstarlab, o=Monstarlab, l=Prague, s=Czech, c=CZ",
+    '-keypass',
+    '123456',
+    '-storepass',
+    '123456',
```

Comment:
needs to be `passwordHash`

Reply:
ah i thought we're doing this for release only?

Reply:
all god now 👍